openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -388,6 +388,12 @@ def user_timezone(self):
         user_timezone_locale = user_timezone_locale_prefs(self.request)
         return user_timezone_locale['user_timezone']
 
+    @property
+    def can_access_proctoring_exams(self):
```

Comment:
naming nit - can_access_proctored_exams or can_access_exam_proctoring

Reply:
Will update it to `can_access_proctored_exams`